test/e2e/scheduling/rescheduler.go

```diff
@@ -0,0 +1,133 @@
+/*
+Copyright 2016 The Kubernetes Authors.
```

Comment:
2018

Reply:
that's a revert PR, let keep it as is :)

Reply:
OK, make sense!